help/en/package/jmri/jmrit/display/MultiSensorIconAdder.shtml

```diff
@@ -34,8 +52,14 @@
       "undefined". When the sensor row is dropped on the label, it will display the sensor
       name.</p>
 
+      <p>The Up &lt;-&gt; Down and Right &lt;-&gt; Left radio buttons determine where the user clicks to 
+      go to the next or previous sensor icon.  If Up &lt;-&gt; Down is selected, clicking in the 
+      top half will advance the selection.  If Right &lt;-&gt; Left is selected, clicking in the 
+      right half will advance the selection.</p>
+      
```

Comment:
(Not sure how to state this...) Using widely-varying wdths and/or heights among the images for a single multisensor can make it harder for the user to understand where the effective "go up" and "go down" (or "go left" and "go right") areas exist.  Icons should (probably) be defined to take up the same area, to make it "more-obvious" where the "go up" and "go down" (or "go left" and "go right") areas exist.

Reply:
Thanks for the suggestion.  I added a sentence to help clarify this.